src/sbt-test/docker/file-permission/changes/strategy-copychown.sbt

```diff
@@ -2,3 +2,7 @@ import com.typesafe.sbt.packager.docker._
 
 dockerPermissionStrategy := DockerPermissionStrategy.CopyChown
 dockerBaseImage          := "fabric8/java-centos-openjdk8-jdk"
+
+// opt-out of numeric USER
+daemonUserUid in Docker  := None
+daemonUser in Docker     := "daemon"
```

Comment:
Shouldn't this be demiourgos728 too?

Reply:
It's demiourgos728 by default, but here I am demonstrating that it can be changed back to daemon if someone is using that from base image etc.